deployments/local-minio/deploy.sh

```diff
@@ -1,64 +1,118 @@
 #!/usr/bin/env bash
 
 # Name to give the docker network and stack. Update network instances in docker-compose.yml file if modified.
-networkname="cdr_local-minio"
+name="cdr_local-minio"
 
+helptext="\
+Usage: deploy.sh [command]
+Utility script for setting up a docker network and deploying a local instance of CDR into a docker swarm.
+'deploy' is the default option if no command is given.
+
+Commands:
+  d, deploy\t\tcreate the docker network and stack, stopping if one already exists
+  u, update\t\tattempts an in-place update of the deployed docker stack
+  c, clean\t\tremoves the deployed docker stack and network (destructive)
+  r, redeploy\t\tclean + deploy: completely removes the stack and network and deploys new instances (destructive)
+"
+
+# Removes the docker stack and network.
+# Docker networks usually take a few seconds to go down, so it waits to prevent redeployment issues
+function cleanup () {
+    printf "Removing $name docker stack and network...\n"
+    docker stack rm $name
+    docker network rm $name
+    waiting=$(docker network ls | grep "$name")
+    printf "\nWaiting for docker network '$name' to go down."
+    while [ ! -z "$waiting" ]; do
+        sleep 2
+        printf "."
+        waiting=$(docker network ls | grep " $name ")
+    done
+    printf "\nDone!\n"
+}
+
+# Waits for everything to start up
 function wait_for_containers () {
-    waiting=$(docker service ls | grep ""$networkname"_.*0/1 ")
-    printf "\nWaiting for docker services to start."
+    waiting=$(docker service ls | grep ""$name"_.*0/1 ")
```

Comment:
If the day comes when there is more than 1 container running for a service, this grep statement will stop working. I think this will work:
`grep $name_.*0/[[:digit:]]+"`